### ISSUE TITLE
About result

### ISSUE THREAD

**Issue by lilala0** [alexandru-dinu/cae#9]
@alexandru-dinu 
Can i ask you if the test result of your trained model reach PSNR of 37 ，SSIM of 0.96 , MS-SSIM of 0.98 at 2bpp as mentioned in the paper “LOSSY IMAGE COMPRESSION WITH COMPRESSIVE AUTOENCODERS”？

**Comment by lilala0** [comment]
@alexandru-dinu 
Excuse me .Can i ask you if the test result of your trained model reach PSNR of 37 ，SSIM of 0.96 , MS-SSIM of 0.98 at 2bpp as mentioned in the paper “LOSSY IMAGE COMPRESSION WITH COMPRESSIVE AUTOENCODERS”

**Comment by alexandru-dinu** [comment]
Hello, @lilala0,

Sorry for the late reply. 

I have not thoroughly tested the models, nor did I perform comparisons with the model described in the mentioned papers. At that time, I have just implemented some ad-hoc models who _seemed_ to perform well, but I did not have time to write an extensive evaluation report.

I will do my best to update the project's wiki this  this kind of information. Also, feel free to create pull requests and/or propose wiki updates. I will greatly appreciate it.

Until further updates, this issue stays open.

Thank you!
